docs/requirements.md

```diff
@@ -11,13 +11,27 @@ The next sections describe in detail how to set up these prerequisites.
 
 The cluster running Codacy must satisfy the following requirements:
 
--   The infrastructure hosting the cluster must be provisioned with the hardware requirements described below
+-   The infrastructure hosting the cluster must be provisioned with the hardware and networking requirements described below
 -   The orchestration platform managing the cluster must be one of:
     -   [Kubernetes](https://kubernetes.io/) **version 1.14.\*** or **1.15.\***
     -   [MicroK8s](https://microk8s.io/) **version 1.15**
 -   Tiller, the server part of [Helm](https://v2.helm.sh/) **version 2.16**, must be installed in the cluster
 -   The [NGINX Ingress controller](https://github.com/kubernetes/ingress-nginx) must be installed and correctly set up in the cluster
 
+### Cluster networking requirements
+
+The cluster must be configured to accept and establish connections on the following ports:
+
+|          | Service      | Protocol/Port | Notes                                                        |
+| -------- | ------------ | ------------- | ------------------------------------------------------------ |
+| Inbound  | SSH          | TCP/22        | Optional, to access the infrastructure remotely.             |
```

Comment:
For "regular" kubernetes it doesn't make sense to put the 22 port. 
We only need kubectl correctly configured.

For microk8s we need the port 22.

Reply:
Changing the notes to "**MicroK8s only**, to access the infrastructure remotely".